packages/web-components/src/components/rux-pop-up-menu/rux-pop-up-menu.scss

```diff
@@ -1,106 +1,53 @@
 :host {
     display: block;
-
-    /**
-      * @prop --popup-menu-background-color: Pop Up Menu Backround Color
-    */
-
-    --popup-menu-background-color: var(--color-background-base-default);
-
-    /**
-      * @prop --popup-menu-border-color: Pop Up Menu Border Color
-    */
-    --popup-menu-border-color: var(--color-border-interactive-muted);
-
-    /**
-      * @prop --popup-menu-caret-background-color: Pop Up Menu Caret Background Color
-    */
-    --popup-menu-caret-background-color: var(--color-border-interactive-muted);
-
-    /**
-      * @prop --popup-menu-caret-left: Position of Caret
-    */
-    --popup-menu-caret-left: 2px;
-
-    /**
-      * @prop --popup-menu-caret-size: Size of Caret
-    */
-    --popup-menu-caret-size: 1.875rem;
-
-    /**
-      * @prop --popup-menu-transition-speed: Transition Time of Pop Up Animation
-    */
-    --popup-menu-transition-speed: 0.1667s;
-
-    opacity: 0;
-    margin: 0;
-    padding: 0;
-    position: absolute;
-    pointer-events: none;
-    color: var(--color-background-interactive-default);
-    background-color: var(--popup-menu-border-color);
-    border: 1px solid var(--popup-menu-border-color);
-    border-top-width: 4px;
-    z-index: 10000;
-    -webkit-user-select: none;
-    -moz-user-select: none;
-    -ms-user-select: none;
-    user-select: none;
-    border-radius: var(--radius-base);
-    top: -9999rem;
-    left: -9999rem;
-    transition: opacity 0.1667s ease-out;
-    filter: drop-shadow(0 2px 4px rgba(0, 0, 0, 0.5));
+    width: min-content;
 }
 
+.hidden,
 :host([hidden]) {
     display: none;
 }
-
-:host([open]) {
-    pointer-events: auto;
-    opacity: 1;
-    transition: opacity 0.1667s ease-in;
+.hidden {
+    //! This allows for the display to be changed with the ::part(popup-content) but still
+    //! be hidden when it needs to be. Any better ideas?
+    display: none !important;
```

Comment:
@markacianfrani if an external dev uses the ::part(popup-content) to change the display, it'll override the .hidden and always show the pop-up. This fixes that, so when .hidden is present the display: none takes precedence. I know using !important is bad form though, so if you can think of a different way I'm all ears